paasta_tools/autoscaling/pause_service_autoscaler.py

```diff
@@ -15,11 +17,11 @@ def get_service_autoscale_pause_time(cluster):
         paasta_print('Could not connect to zookeeper server')
         return 2
 
-    pause_time = int(pause_time)
+    pause_time = float(pause_time)
     if pause_time < time.time():
         paasta_print('Service autoscaler is not paused')
     else:
-        paused_readable = datetime.fromtimestamp(pause_time).strftime('%F %H:%M:%S UTC')
+        paused_readable = datetime.fromtimestamp(pause_time, tz=pytz.timezone('America/Los_Angeles')).strftime('%F %H:%M:%S YST')  # NOQA
```

Comment:
We could do something like:

from tzlocal import get_localzone
local_tz = get_localzone()
paused_readable = local_tz.localize(datetime.fromtimestamp((pause_time)))


However it will require to add the module tzlocal in requierements.txt